testing/src/main/java/org/oppia/android/testing/TextInputAction.kt

```diff
@@ -0,0 +1,47 @@
+package org.oppia.android.testing
+
+import android.content.Context
+import android.view.View
+import androidx.annotation.StringRes
+import com.google.android.material.textfield.TextInputLayout
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+import javax.inject.Inject
+
+/**
+ * Action for accessing HasErrorTest and HasNoErrorTest Functions.
+ *
+ * This is needed because Robolectric doesn't seem to properly input digits for text views using
+ * 'android:digits' or other filters. See https://github.com/robolectric/robolectric/issues/5110
```

Comment:
This doesn't sound correct to me, because `TextInputAction` is all about testing `TextInputLayout` and `TextInputEditText`. Nothing is related to `android:digits`. 

> Action for accessing HasErrorTest and HasNoErrorTest Functions.

This gives a sense of the function names inside the class but not when to use this class.

Reply:
/**
 *  This class is used in testing of TextInputLayout and TextInputEditText.
 */
Will it be good

Reply:
```kotlin
/**
  * Action for matching error text and performing other actions on TextInptLayout in a test infrastructure-specific way.
  */
```

Reply:
Use this above class

Reply:
ok has added it